src/layouts/MainLayout.vue

```diff
@@ -139,14 +139,17 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+//@dummy genesis gradient = background: linear-gradient(#252a5e 27.19%, #2d4684 65.83%);
```

Comment:
remove comment if not applying rule

Reply:
removed comment:
//@dummy genesis gradient = background: linear-gradient(#252a5e 27.19%, #2d4684 65.83%);